domain/schema/controller/sql/0009-controller-node.sql

```diff
@@ -9,3 +9,15 @@ ON controller_node (dqlite_node_id);
 
 CREATE UNIQUE INDEX idx_controller_node_bind_address
 ON controller_node (bind_address);
+
+-- controller_node_agent_version tracks the reported agent version running for
+-- each controller in the cluster.
+CREATE TABLE controller_node_agent_version (
+    controller_id TEXT NOT NULL PRIMARY KEY,
+    version TEXT NOT NULL,
+    architecture TEXT NOT NULL,
+    release TEXT NOT NULL,
```

Comment:
The old Tools `Release` attribute used to be for recording the OS, eg ubuntu, windows, centos etc.
1. It's poorly named
2. We don't need it any more since we just want to build single agent binary for "linux". I am pretty sure Release is effectively hard coded to "ubuntu" at the moment. We should just drop this attribute until we have a concrete need for it, which I'm not expecting we will.

Reply:
Done & Done